code/datums/spells/bloodcrawl.dm

```diff
@@ -149,9 +149,13 @@
 		var/mob/living/simple_animal/demon/slaughter/demon = L
 		demon.devoured++
 		to_chat(victim, "<span class='userdanger'>You feel teeth sink into your flesh, and the--</span>")
+		var/obj/item/organ/internal/regenerative_core/legion/core = victim.get_int_organ(/obj/item/organ/internal/regenerative_core/legion)
+		if(core)
+			core.remove(victim)
```

Comment:
Have you ensured it does not drop somewhere when doing this? Might drop inside the demon, unsure what happens in that situation.

Reply:
yeah, you might want to `qdel()` it here after removal

Reply:
Mirror of what I said in coderchat. Testing doesn't seem to drop the core, however added a safety qdel(core) anyway just in case its dropping it in the demon's level.